docs/user-guide/recipes.rst

```diff
@@ -38,11 +38,18 @@ Full documentation for /etc/crane.conf can be found in the Crane `README`_.
 Sync
 ----
 
-The pulp-docker plugin supports synchronizing from upstream repositories as of
-version 0.2.1. As of version 2.0.0, it can synchronize with either Docker v1 or
-v2 registries.
+The pulp-docker plugin supports synchronizing from docker V2 API compatible registries.
 
+<<<<<<< HEAD
```

Comment:
I think the conflict was not resolved here.

Reply:
fixed.